libraries/eddystone-advertising-library/bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "eddystone-advertising-library",
+  "version": "0.3",
+  "homepage": "https://github.com/google/eddystone/tree/master/libraries/eddystone-advertising-library",
```

Comment:
ah, i see. well in that case let's s/chromeos/javascript, or web-js, or whatever makes sense there. i'd just like there to be one level of something that clearly identifies what it is because "eddystone-advertising-library" is very generic (and i can imagine there being more than one library)